builtin/providers/google/resource_compute_region_instance_group_manager.go

```diff
@@ -0,0 +1,420 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+)
+
+func resourceComputeRegionInstanceGroupManager() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeRegionInstanceGroupManagerCreate,
+		Read:   resourceComputeRegionInstanceGroupManagerRead,
+		Update: resourceComputeRegionInstanceGroupManagerUpdate,
+		Delete: resourceComputeRegionInstanceGroupManagerDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
```

Comment:
I'd love to see an Exists function here, too, just for completeness. Though kudos on the Import support right out the gate!